content/ges/0002-organizacoes-recursos-humanos.md

```diff
@@ -96,7 +96,7 @@ Presidente da República > Primeiro Ministro > Ministros > Secretários > Tu.
 
    - **Vantagens**: Especialização dos empregados. Economias de Escala.
    - **Desvantagens**: Fraca comunicação dos departamentos.
-   - **Características**: Estabilidade, Eficiência e Empresas de grande Dimensão.
+   - **Características**: Estabilidade, Eficiência em Empresas de grande Dimensão.
```

Comment:
I'm not sure if this is correct, from my POV this means "normally used by big companies", which was what the previous content meant.
@randomicecube what do you think?

Reply:
Agreed, I think the previous content was correct.